src/content/staking/pools/index.md

```diff
@@ -72,6 +72,14 @@ Currently, withdrawing funds from a validator on the Beacon Chain is not possibl
 Alternatively, pools that utilize an ERC-20 liquidity token allow users to trade this token in the open market, effectively allowing you to "withdraw" without actually removing ETH from the Beacon Chain.
 </ExpandableCard>
 
+<ExpandableCard title="I am a pooled staker, do I need to do anything for The Merge?">
+ETH/token holders, pooled/SaaS stakers and general Ethereum users <strong>do not need to do anything to prepare for The Merge</strong>.
+
+As The Merge approaches, be on high alert for scammers. You do not need to do anything to "upgrade" your ETH or staked ETH tokens for the transition to proof-of-stake.
```

Comment:
Removed 'do anything' as it implies that some 'upgrade' happens in the background.

```suggestion
However, as The Merge approaches, be on high alert for scammers. **You do not need to upgrade your ETH or staked ETH tokens** for the transition to proof-of-stake.
```

Reply:
?

Reply:
Soz, my explanation was poorly worded @wackerow 

> You don't need to do anything to upgrade your ETH.

Implies that your ETH will be upgraded, but you don't need to act to make this happen.

> You don't need to upgrade your ETH.

States that your ETH doesn't need to be upgraded.